intents/main.py

```diff
@@ -18,10 +18,10 @@
 It returns the result of the intent and also the list of suggestions.
 """
 
-import json, pandas, enum, topk, slice_compare
+import json, pandas, enum, topk, slice_compare, correlation, trend, time_compare
 from flask import escape
```

Comment:
Correlation Is not yet submitted to the Googleinterns:master . This change should be done after I submit correlation.py